core_api/src/build_chains.py

```diff
@@ -90,7 +89,7 @@ def make_document_context(input_dict):
             documents += chunks
 
         # right now, can only handle a single document so we manually truncate
-        max_tokens = 20_000  # parameterise later
+        max_tokens = (env.ai.max_tokens,)
```

Comment:
Should we maybe call this something more specific? env.ai.summary_map_chunk_size?